index.html

```diff
@@ -16,21 +16,73 @@
     <title>White Flag Project</title>
+    
+    <!-- Vavilon.js for i18n -->
+    <script
+      src="static/i18n/ms.json"
+      type="application/json"
+      data-vavilon-dict="ms"
+    ></script>
+    <script
+      src="static/i18n/ta.json"
+      type="application/json"
+      data-vavilon-dict="ta"
+    ></script>
+    <script
+      src="static/i18n/zh.json"
+      type="application/json"
+      data-vavilon-dict="zh"
+    ></script>
+    <script src="https://cdn.jsdelivr.net/npm/vavilon@1.0.0/dist/vavilon.min.js"></script>
 </head>
 <body>
-
-     <!-- main container -->
-        <!-- nav menu split into 2 for flex usage -->
-        <div class="nav-menu">
-            <div class="image-logo">
-                <img src="static/images/white flag logo.svg" alt="white flag project logo" />
-            </div>
-            <div class="nav-content">
-                <p>How can you help?</p>
-                <a href="https://discord.com/invite/cNaZPAEk"><img src="static/images/join_our_discord.svg" alt="Join our Discord" /></a>
-
-            </div>
-
+    <!-- main container -->
+    <!-- nav menu split into 2 for flex usage -->
+    <div class="nav-menu">
+        <div class="image-logo">
+            <img src="static/images/white flag logo.svg" alt="white flag project logo" />
+        </div>
+        <div class="nav-content">
+        <p class="vavilon" data-vavilon="nav-help">How can you help?</p>
+        <a href="https://discord.com/invite/cNaZPAEk"
+          ><img src="static/images/join_our_discord.svg" alt="Join our Discord"
+        /></a>
+
+        <!-- to enable language switching -->
+        <script>
+          function toggleDropdown() {
+            document
+              .getElementById("dropdown-content")
+              .classList.toggle("show");
+          }
+
+          window.onclick = function (event) {
+            if (!event.target.parentElement.matches(".lang-dropdownbtn")) {
+              var dropdown = document.getElementById("dropdown-content");
+              if (dropdown.classList.contains("show")) {
+                dropdown.classList.remove("show");
+              }
+            } else {
+            }
+          };
+        </script>
+
+        <div class="lang-dropdown">
+          <button
+            class="lang-dropdownbtn is-hoverable is-clickable"
+            onclick="toggleDropdown()"
+          >
+            <p class="vavilon" data-vavilon="current-language">English</p>
+            <i class="dropdown-arrow"> </i>
+          </button>
+          <div id="dropdown-content" class="lang-options">
+            <a onclick="setLang('en')">English</a>
+            <a onclick="setLang('ms')">Malay</a>
+            <a onclick="setLang('zh')">Mandarin</a>
+            <a onclick="setLang('ta')">Tamil</a>
```

Comment:
Very nitpicky, but I feel like we should display the name of the language in the language itself:

```suggestion
            <a onclick="setLang('en')">English</a>
            <a onclick="setLang('ms')">Bahasa Malaysia</a>
            <a onclick="setLang('zh')">中文</a>
            <a onclick="setLang('ta')">தமிழ்</a>
```